app/[locale]/history/page.tsx

```diff
@@ -1,5 +1,199 @@
+    method: 'POST',
+    full_url: 'http://localhost:3000/POST/...',
+    url: 'https://api.example.com/users',
+    latency: '340ms',
+    status: 201,
+    timestamp: '2025-09-09 10:20:05',
+    requestSize: '1.2KB',
+    responseSize: '450B',
+    error: null,
+  },
+  {
+    idHistory: '3',
+    method: 'PUT',
+    full_url: 'http://localhost:3000/PUT/...',
+    url: 'https://api.example.com/users/42',
+    latency: '280ms',
+    status: 200,
+    timestamp: '2025-09-09 10:35:48',
+    requestSize: '1KB',
+    responseSize: '1.1KB',
+    error: null,
+  },
+  {
+    idHistory: '4',
+    method: 'DELETE',
+    full_url: 'http://localhost:3000/DELETE/...',
+    url: 'https://api.example.com/users/99',
+    latency: '150ms',
+    status: 404,
+    timestamp: '2025-09-09 11:00:12',
+    requestSize: '220B',
+    responseSize: '300B',
+    error: 'Not Found',
+  },
+  {
+    idHistory: '5',
+    method: 'PATCH',
+    full_url: 'http://localhost:3000/PATCH/...',
+    url: 'https://api.example.com/reports',
+    latency: '700ms',
+    status: 500,
+    timestamp: '2025-09-09 11:05:41',
+    requestSize: '400B',
+    responseSize: '0B',
+    error: 'Internal Server Error',
+  },
+  {
+    idHistory: '6',
+    method: 'HEAD',
+    full_url: 'http://localhost:3000/HEAD/...',
+    url: 'https://api.example.com/reports',
+    latency: '700ms',
+    status: 500,
+    timestamp: '2025-09-09 11:05:41',
+    requestSize: '400B',
+    responseSize: '0B',
+    error: 'Internal Server Error',
+  },
+  {
+    idHistory: '7',
+    method: 'OPTIONS',
+    full_url: 'http://localhost:3000/OPTIONS/...',
+    url: 'https://api.example.com/reports',
+    latency: '700ms',
+    status: 500,
+    timestamp: '2025-09-09 11:05:41',
+    requestSize: '400B',
+    responseSize: '0B',
+    error: 'Internal Server Error',
+  },
+];
+
 export default function History() {
+  const t = useTranslations('history');
+  const HEADERS = [
+    t('method'),
+    t('url'),
+    t('link'),
+    t('status'),
+    t('latency'),
+    t('timestamp'),
+    t('requestSize'),
+    t('responseSize'),
+    t('error'),
+  ];
+
+  if (!mockResponseHistory.length) {
+    return (
+      <div className="flex flex-col justify-center mx-auto">
+        <p className="text-xl font-semibold tracking-tight pb-10">
+          {t('titleNotFound')}:
+        </p>
+        <Button className="mx-auto">
+          <Link href="/rest-client">{t('buttonREST')}</Link>
+        </Button>
+      </div>
+    );
+  }
+
   return (
-    <h1>History & Analytics (private, lazy-loaded, server-side generated)</h1>
+    <div className="flex flex-col justify-center w-full">
+      <h2 className="text-2xl font-semibold tracking-tight first:mt-0 pb-8">
+        {t('title')}
+      </h2>
+      <Table>
+        <TableHeader>
+          <TableRow>
+            {HEADERS.map((head, index) => (
+              <TableHead className="w-[100px]" key={index}>
+                {head}
+              </TableHead>
+            ))}
```

Comment:
```suggestion
export const HEADERS = [
  'method',
  'url',
  'link',
  'status',
  'latency',
  'timestamp',
  'requestSize',
  'responseSize',
  'error',
];

{HEADERS.map((head, index) => (
  <TableHead className="w-[100px]" key={index}>
    {t(head)}
  </TableHead>
 ))}
```

Так получится массив убрать из компонента

Reply:
поправила